merlin_standard_lib/schema/schema.py

```diff
@@ -103,7 +102,7 @@ def create_continuous(
         tags: Optional[TagsType] = None,
         **kwargs,
     ) -> "ColumnSchema":
-        _tags: List[str] = [str(t) for t in tags] if tags else []
+        _tags: List[str] = [t.value for t in TagSet(tags or [])]
```

Comment:
This is something I don't love - `_tags` is a `List[str]` not a `TagsType`. We convert actual `Tags` to their string value, but I don't think we convert them back anywhere. I need to understand it a little more now that the tests are passing (hopefully)

Reply:
I agree. I think we converted to strings everywhere to make it easier for users to provide custom tags as strings. Not completely sure how we handle that currently across Merlin.